bin/cml/runner.js

```diff
@@ -270,47 +272,55 @@ const runLocal = async (opts) => {
   RUNNER = proc;
   ({ id: RUNNER_ID } = await cml.runnerByName({ name }));
 
-  if (parseInt(idleTimeout) > 0) {
+  if (idleTimeout > 0) {
```

Comment:
Correct in assuming that https://github.com/iterative/cml/pull/882 has made the remove `parseInt` redundant?